mfthomps/CyberCIEGE-port-to-Unity3d
# issue/239-character-prefab-refactor

Closes #239 

This creates new character prefabs and variants to hopefully make it easier to manage all the different varieties. 

* Created base prefabs with common used functionality
* Created variants with their specialties.

To verify:
- Run any scenario and verify characters operate the same
  - Doors open/close for them
  - Players can click on them
  - They stop moving/animating when the game is paused
  - The Users wander if they can't work. The IT and Security wander all the time.
